### ISSUE TITLE
Continue training bug

### ISSUE THREAD

**Issue by jielu0728** [ultralytics/yolov5#284]
I've trained my model for 50 epochs, the best stopped at 30th epoch.
When I continue training using the best weights with train.py, I observe that the learning rate of the first (31st) epoch is reset to the initial learning rate (0.01) which is supposed to be 0.0025+ because of lr scheduler. 
However when 32nd epoch starts, learning rate returns to normal (about 0.0023).

**Comment by glenn-jocher** [comment]
@jielu0728 thanks for the notice. Could you try some fixes and submit a PR for what works best? The LR scheduler code is here:

https://github.com/ultralytics/yolov5/blob/3bdea3f697d4fce36c8e24a0701c0f419fa8f63a/train.py#L148-L154
